cli/command/service/opts.go

```diff
@@ -510,7 +510,7 @@ func addServiceFlags(cmd *cobra.Command, opts *serviceOptions) {
 	flags.Var(&opts.restartPolicy.window, flagRestartWindow, "Window used to evaluate the restart policy")
 
 	flags.Uint64Var(&opts.update.parallelism, flagUpdateParallelism, 1, "Maximum number of tasks updated simultaneously (0 to update all at once)")
-	flags.DurationVar(&opts.update.delay, flagUpdateDelay, time.Duration(0), "Delay between updates")
+	flags.DurationVar(&opts.update.delay, flagUpdateDelay, time.Duration(0), "Delay between updates (Default none and Valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h')")
```

Comment:
@milindchawre it's awesome to see you addressing this issue! 👌
Looking at the existing descriptions I think you should switch the order of the description to be

```
<description> <valid-opt1|valid-opt2|...> <default>
```

As an example if you look at the `--update-failure-action` flag

```
--update-failure-action string   Action on update failure (pause|continue) (default "pause")
```

This format also ensures the description is short. This is important, otherwise the line will wrap around and makes it really hard to read the descriptions.